openadapt/config.py

```diff
@@ -119,7 +137,9 @@ def getenv_fallback(var_name):
 ROOT_DIRPATH = pathlib.Path(__file__).parent.parent.resolve()
 DB_FPATH = ROOT_DIRPATH / DB_FNAME  # type: ignore # noqa
 DB_URL = f"sqlite:///{DB_FPATH}"
+DT_FMT = "%Y-%m-%d_%H-%M-%S"
 DIRNAME_PERFORMANCE_PLOTS = "performance"
+ZIPPED_RECORDING_FOLDER_PATH = ROOT_DIRPATH / "data" / "zipped"
```

Comment:
Please add:

```
DATA_DIRECTORY_PATH = ROOT_DIRPATH / "data"
RECORDING_DIRECTORY_PATH = DATA_DIRECTORY_PATH / "recordings"
```

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/223